.storybook/reset.scss

```diff
@@ -11,6 +15,22 @@ body,
 body {
   margin: 0;
   padding: 0 !important;
-  font-family: "IBM Plex Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+// TYPOGRAPHY
+
+// From normalize.css
+// https://github.com/necolas/normalize.css/blob/fc091cce1534909334c1911709a39c22d406977b/normalize.css#L160-L165
+button,
+input,
+select,
+textarea {
+  font-family: inherit;
+}
+
+body {
   color: $color--black;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: font-scale();
+  line-height: line-height-scale();
```

Comment:
I think the line-height changed quite a lot in screener, for example this one https://screener.io/v2/states/H1OkdIZtV.acl-services-paprika/BOT-2134--remove-lato-from-components/1280x1024/Chrome/sortableautomation-tests-screener-0
is it ok?

Reply:
I'm okay with it. This should now more accurately reflect our apps (they should have a default line-height of ~1.4).  In that specific case, only the dummy content is affected.